src/main/cms/cms_menu_imu.c

```diff
@@ -101,7 +104,10 @@ static const char * const cms_mixerImplTypeLabels[] = {
 };
 
 static const char * const cms_FilterType[] = {
-    "PT1", "BIQUAD", "PT2", "PT3", "PT4",
+    "PT1", "BIQUAD", "PT2", "PT3", "PT4"
```

Comment:
You should leave the trailing comma, is there for a reason (explained below) :)

Reply:
Apologies, will fix